### PR TITLE
UCP/API: fixed typo in tag_recv_nbx

### PR DESCRIPTION
**Pull request by Sergei-Lebedev**
## What
fixed typo in tag_recv_nbx description

### CODE REVIEW

**Review by brminich — DISMISSED** [review]

**Review by yosefe — DISMISSED** [review]

**Comment by shamisp** [comment]
👍 
@tonycurtis  pls take a look

File: src/ucp/api/ucp.h [hunk 1]
```diff
@@ -3490,7 +3490,7 @@ ucs_status_ptr_t ucp_stream_recv_data_nb(ucp_ep_h ep, size_t *length);
  * buffer.  In order to notify the application about completion of the receive
  * operation the UCP library will invoke the call-back @a cb when the received
  * message is in the receive buffer and ready for application access.  If the
- * receive operation cannot be stated the routine returns an error.
+ * receive operation cannot be started the routine returns an error.
```
Comment (tonycurtis) [inline src/ucp/api/ucp.h:3493]: think "started, then the ..." reads better

**Review by tonycurtis — CHANGES_REQUESTED** [review]

File: src/ucp/api/ucp.h (hunk 1 above)
Comment (Sergei-Lebedev) [reply src/ucp/api/ucp.h]: done, also fixed for ucp_tag_send_sync_nbx, ucp_stream_recv_data_nb

**Review by yosefe — APPROVED** [review]

**Comment by yosefe** [comment]
@tonycurtis @shamisp is this ok now?

**Comment by shamisp** [comment]
Looks good to me

**Comment by tonycurtis** [comment]
Looks good, thanks

**Pull request merged**